webapp/pages/__tests__/Dashboard.spec.jsx

```diff
@@ -0,0 +1,62 @@
+import xhrmock from 'xhr-mock';
+import React from 'react';
+import {mount} from 'enzyme';
+
+import Dashboard from '../Dashboard';
+import {RepositoryContext} from '../App';
+
+describe('Dashboard', () => {
+  // replace the real XHR object with the mock XHR object before each test
+  beforeEach(() => xhrmock.setup());
+
+  // put the real XHR object back and clear the mocks after each test
+  afterEach(() => xhrmock.teardown());
+
+  it('renders with repos loaded', () => {
+    let repo = TestStubs.Repository();
+    let store = TestStubs.store({
+      repos: {
+        items: [repo],
+        loading: false
+      }
+    });
+    let context = TestStubs.standardContext({
+      store
+    });
+
+    const wrapper = mount(
+      <RepositoryContext repoList={[repo]} loading={false}>
```

Comment:
had to do this as passing the context directly doesn't work due to the wrapper component not handling childContextTypes